src/mediaelement.js

```diff
@@ -160,7 +160,7 @@ export default class MediaElement extends WebAudio {
         elt.controls = this.params.mediaControls;
         elt.autoplay = this.params.autoplay || false;
 
-        this._load(elt, peaks);
+        this._load(elt, peaks, elt.preload);
```

Comment:
What if `elt.preload` is undefined?

Reply:
good question. 

It won't be undefined.  If you don't set the HTML tag, in chrome/ff, the `preload` element will default to `""`.  In Safari, it defaults to `"auto"`.  The behavior is also similar: chrome/ff don't preload at all, and Safari first does a range request and then ends up preloading a whole 2mb file. 

I can detect these cases here and change our `.load` behavior accordingly, but I'm really not sure what the user's intention really is.  My guess is that if they're providing peak data they almost *never* want a preload, but that's just a guess...